packages/design-system/src/components/category_dropdown/CategoryDropDown.tsx

```diff
@@ -45,7 +45,7 @@ const CategoryDropDown = ({ size, categories }: CategoryDropDownProps) => {
     <div className={categoryDropdownVariants({ size })}>
       <div
         onClick={() => setIsDropDownOpen(!isDropDownOpen)}
```

Comment:
```suggestion
onClick={() => setIsDropDownOpen((prev) => !prev)}
```

물론 현재 코드에서는 크게 문제 될 경우는 없지만 상태 값을 변경할 때는 `isDropDownOpen`로 직접 변경하는 것보다 `useState`에서 제공하는 `prev` 값을 사용하는 것이 더 안정적인 방법이에요! 

**[ 공식 문서 참고 ]**
https://ko.react.dev/reference/react/useState#updating-state-based-on-the-previous-stat

Reply:
확인하였습니다! 적용해두었습니다